pennylane/gradients/__init__.py

```diff
@@ -50,6 +50,17 @@
     hadamard_grad
     stoch_pulse_grad
     pulse_odegen
+    classical_jacobian
+
+Metric tensors
+^^^^^^^^^^^^^^
+
+.. autosummary::
+    :toctree: api
+
+    metric_tensor
+    adjoint_metric_tensor

```

Comment:
```suggestion
.. currentmodule:: pennylane

.. autosummary::
    :toctree: api

    metric_tensor
    adjoint_metric_tensor

.. currentmodule:: pennylane.gradients
```

Should fix it I hope 🤞

Reply:
@rmoyard @trbromley I would say this is a blocking comment. I think it's very important that the metric tensor does not appear in the documentation twice (as `qml.metric_tensor` and `qml.transforms.metric_tensor`) for a couple of reasons:

- Biggest reason: Suddenly, `qml.gradients.metric_tensor` is no longer an implementation detail -- it will look like this is our recommendation for how to call it, for users who end up on this page!

- It will cause issues with cross referencing. `:func:.metric_tensor` will no longer resolve, as Sphinx won't know which one to link to. The cross referencing will silently fail, and no link will be generated. This will affect both the PL docs _and_ the demos!

- It will cause issues with SEO and analytics, since both pages will show up in google

Reply:
Thanks Josh, it makes senses!

Reply:
Is there still something blocking @josh146 ?